frontend/src/API/Interfaces.ts

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+export interface DoctorI {
+  id: number;
+  name: string;
+  surname: string;
+  specialist: string;
+}
+export interface PatientI {
+  id: number;
+  name: string;
+  surname: string;
+  insured: boolean;
+}
+export interface AppointmentI {
+  id: number;
+  patient: PatientI;
+  doctor: DoctorI;
+  cost: number;
+  appointmentDate: number; // LocalDateTime ??
+}
+export interface MedicalFacilityI {
+  id: number;
+  name: string;
+  localization: string;
+  doctorCount: number;
+  patientCount: number;
+  contactNumber: number;
+}
+export interface PatientMedicalDataI {
+  id: number;
+  patient: PatientI;
+  doctor: DoctorI;
+  treatmentDate: number; // LocalDateTime ??
```

Comment:
We can pass smth like "appointmentDate":"2014-04-28T16:00:49.455" via JSON. Should be then string?